src/ex6/client/src/App.css

```diff
@@ -0,0 +1,26 @@
+.todoApp {
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  transition: 0.3s;
+  width: 85%;
+  margin: auto;
+  width: 30%;
+  border: 3px solid rgb(0, 62, 128);
+  padding: 10px;
+  background-color: rgb(255, 255, 255);
+  overflow: clip;
+  box-sizing: border-box;
+  font-family: "Lucida Console", "Courier New", monospace;
+  font-family:"Franklin Gothic Medium";
```

Comment:
having the same property twice causes redundancy/conflicts
choose one and possibly use [font fallbacks](https://www.w3schools.com/css/css_font_fallbacks.asp) if needed